examples/ex2/run.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+source ../../config.sh
+OS_NAME=$(uname)
+
+# Delete old compiled files
+\rm -f exe
+\rm -f *.o
+
+#runtime dynamic library path
+RPATH="$(pwd)/../.."
```

Comment:
Here we need to append the path of FFNN library to avoid compilation fail on linux

Reply:
I will push this quickly..